pkg/kubelet/pod_workers.go

```diff
@@ -765,12 +797,18 @@ func (p *podWorkers) managePodLoop(podUpdates <-chan podWork) {
 			klog.ErrorS(err, "Error syncing pod, skipping", "pod", klog.KObj(pod), "podUID", pod.UID)
 
 		case update.WorkType == TerminatedPodWork:
+			klog.V(4).InfoS("Processing pod event done", "pod", klog.KObj(pod), "podUID", pod.UID, "updateType", update.WorkType)
```

Comment:
The processing hasn't completed yet, so this log becomes potentially quite misleading.